hermeznetwork/bridge-ui
# Add GET /bridges/{ethereumAddress} API call

Closes #26.

### What does this PR does?

This PR adds the necessary code to retrieve the `bridges` from the API. It also adds `axios` to the `fiat-exchange-rates-api` module.

### How to test?

First of all you'll need to deploy the Bridge API in your machine using Docker: https://github.com/hermeznetwork/hermez-bridge

After that, you can add this lines to the home view and check if the response is properly logged into the console.

```
const { getBridges } = useBridgeContext();

useEffect(() => {
  getBridges({ ethereumAddress: "0xeB17ce701E9D92724AA2ABAdA7E4B28830597Dd9" })
    .then(console.log)
    .catch(console.error);
}, [getBridges]);
```
## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
